server/utils/utils.ts

```diff
@@ -119,3 +121,27 @@ export function mergeDeep(...objects: Record<string, any>[]): Record<string, any
     return newObj
   }, {})
 }
+
+/*
+  Depending on status, steps should be sorted by status, with the ones that have been updated being moved to the bottom of that sorted section.
+  See the Statuses - in order.png for expected ordering.
```

Comment:
maybe a link to the jira ticket or figma would be better than just the image filename?

Reply:
I think there might be concerns about posting links to internal jira/confluence/tooling on the public repos. If the comment is clear enough, we should be okay here 😄

Reply:
the ID of the jira ticket would be enough here - something to give an idea of where the referenced image is.

Reply:
Updated in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/178/commits/285981fb2a55dad2eb089f28cb993c126cde65c3